ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -51,43 +51,76 @@
-                      ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
-                <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode"
-                        asp-items="Model.TBServices" v-on:change="function (ev) {fetchHospitalList(ev); validate('TBServiceCode')(ev);}">
-                </select>
-            </nhs-form-group>
-            <br />
-            @{
-                var HospitalError = !Model.IsValid("Episode.HospitalId");
-                var HospitalGroupState = HospitalError ? Error : Standard;
-                var HospitalSelectErrorState = HospitalError ? SelectType.Error : SelectType.Standard;
-            }
-            <nhs-form-group ref="formGroupHospitalId" nhs-form-group-type=@HospitalGroupState>
-                <label nhs-label-type="Standard" asp-for="Episode.HospitalId">
-                    Hospital (required)
-                </label>
-                <span nhs-span-type="ErrorMessage" id="hospital-error" has-error="@HospitalError"
-                      ref="errorFieldHospitalId" asp-validation-for="Episode.HospitalId"></span>
-                <select ref="selectFieldHospitalId" nhs-select-type="@HospitalSelectErrorState" asp-for="Episode.HospitalId"
-                        asp-items="Model.Hospitals" v-on:change="function(ev) { validate('HospitalId')(ev); }">
-                    <option value="" selected>Select Hospital</option>
-                </select>
-            </nhs-form-group>
-            <br />
-        </div>
-    </cascading-dropdowns>
+
+    @if(Model.Notification.NotificationStatus == NotificationStatus.Draft) 
+    {
+        <cascading-dropdowns model="Episode" shouldvalidatefull="@fullValidation" inline-template>
+            <div>
+                @{
+                    var TBServiceCodeError = !Model.IsValid("Episode.TBServiceCode");
+                    var TBServiceCodeGroupState = TBServiceCodeError ? Error : Standard;
+                    var TBServiceCodeSelectErrorState = TBServiceCodeError ? SelectType.Error : SelectType.Standard;
+                }
+                <nhs-form-group ref="formGroupTBServiceCode" nhs-form-group-type=@TBServiceCodeGroupState>
+                    <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">
+                        Notifying TB service (required)
+                    </label>
+                    <span nhs-span-type="ErrorMessage" id="tb-service-error" has-error="@TBServiceCodeError"
+                        ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
+                    <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode"
+                            asp-items="Model.TBServices" v-on:change="function (ev) {fetchHospitalList(ev); validate('TBServiceCode')(ev);}">

```

Comment:
the nested @{} blocks compile fine but do make VS code complain a fair amount :(